### ISSUE TITLE
Accessibility Audit: Support admin user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5772]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the support admin user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/admin/*`

Use cases to test:
* edit or verify org identity
* add org admin
* create new device
* manage devices
* org data access

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.
